ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -50,6 +69,9 @@ QtObject {
 
     readonly property var sectionDetails: d.sectionDetailsInstantiator.count ? d.sectionDetailsInstantiator.objectAt(0) : null
 
+    // TODO: Review if `mainModuleInst.activeSection` is indeed the same as `sectionDetails` here. If yes, this is redundant
+    readonly property bool joined: root.mainModuleInst.activeSection.joined
```

Comment:
This Store is created for each Chat I think. So it's weird that `joined` here would be using the ActiveSection and not the section it is from.

So this should probably be `chatCommunitySectionModule.joined`

Reply:
That's right! I'll check it again. I just moved this `rootStore.mainModuleInst.activeSection.joined` to `Chat.RootStore` and added a `to-do` to check it again, since it seemed strange to me too.

As you say I think it should be `chatCommunitySectionModule.joined` but I need to check all `MessageView` usages since maybe there are some corner cases to consider, like maybe `CommunityMemberMessagesPopup`.